datastore/emulator.go

```diff
@@ -104,3 +116,31 @@ func Emulator(ctx context.Context) <-chan struct{} {
 
 	return quit
 }
+
+func findEmulatorPath() string {
+	gcloudPath, err := exec.LookPath("gcloud")
+	if err != nil {
+		log.Fatalf("Couldn't determine gcloud path")
+	}
+
+	emulatorPath := strings.Replace(gcloudPath, "bin/gcloud", "platform/cloud-datastore-emulator/cloud_datastore_emulator", 1)
```

Comment:
Go style encourages inlining instead of creating single-use variables.

```
return strings.Replace(gcloudPath, "bin/gcloud", "platform/cloud-datastore-emulator/cloud_datastore_emulator", 1)
```

Reply:
Done